layouts/shortcodes/about/action-link.html

```diff
@@ -0,0 +1,22 @@
+{{- $url := "" -}}
+{{- with .Get "href" }}{{ $url = . }}{{ end -}}
```

Comment:
You can use the [default](https://gohugo.io/functions/default/) function to simplify this.

Reply:
Is this what you have in mind? https://github.com/spotlightpa/poor-richard/pull/338/commits/a36a8d81211abb66705dc1c8ef27e4169333c703

`{{- $url := .Get "href" | default "#" -}}`

Reply:
Yup.